README.md

```diff
@@ -103,10 +103,18 @@ We recommend that pull requests follow this convention:
 ## Limitations
 The limitations of the API are:
 - The application cannot accommodate millions of users for now because the database only allow limited number of users since it is a free account.
-- Users cannot share documents with other users through social sharing
 
 ## FAQ
-- If there are any question you want to ask, contact me via mail [itua.akhator@andela.com](mailto:itua.akhator@andela.com)
+- Is there any monetary value attached before I can use the API ?
+  * No, its absolutely free.
+
+- How do I target APIs endpoints?
+  * You can target the API using ```http://doksmanager.herokuapp.com/api/v1/{route}```
+  you can go to the [documentation](http://doksmanager.herokuapp.com) for more on how to use it.
```

Comment:
You can stick to '**you can go to the [documentation](http://doksmanager.herokuapp.com) for more on how to use it.**' over **You can target the API using ```http://doksmanager.herokuapp.com/api/v1/{route}**
 +